cli/src/opts/wallet/multi_wallet.rs

```diff
@@ -292,7 +331,7 @@ impl MultiWallet {
                 self.keystore_password_files.clone().unwrap_or_default().into_iter();
 
             for path in keystore_paths {
-                wallets.push(self.get_from_keystore(Some(path), passwords_iter.next().as_ref(), password_files_iter.next().as_ref())?.wrap_err("Keystore paths do not have the same length as provided passwords or password files.")?);
+                wallets.push(self.get_from_keystore(Some(&path), passwords_iter.next().as_ref(), password_files_iter.next().as_ref())?.wrap_err("Keystore paths do not have the same length as provided passwords or password files.")?);
```

Comment:
please do this in two steps, the large push( argument makes this harder to read

Reply:
Done